packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -0,0 +1,47 @@
+export default function globalAssignmentToDefaultExport({source}, {jscodeshift: j}, {printOptions = {}, appGlobalIdentifiers}) {
+  function findFirstMember(memberExpression) {
+    if (j.MemberExpression.check(memberExpression)) {
+      return findFirstMember(memberExpression.object);
```

Comment:
I know in Sello we did a lot of `Sello.Components.Component = Component`. Do we care about supporting the multiple nested namespaces?